packages/bpk-docs/src/pages/BadgePage/badge-page.scss

```diff
@@ -20,13 +20,24 @@
 
 .bpkdocs-badge-page {
   &__badge-spacing {
+    max-width: 100%;
+    margin-top: $bpk-spacing-sm;
     margin-right: $bpk-spacing-sm;
+    overflow: hidden;
   }
 
-  &__badge-container {
+  &__badge-container-docked {
     position: relative;
     display: block;
     height: $bpk-one-pixel-rem * 60;
     background-color: $bpk-color-gray-50;
   }
+
+  &__badge-layout-light {
+    padding-top: $bpk-spacing-sm;
+  }
+
+  &__badge-layout-dark {
```

Comment:
Yeah it's basically because BadgeLayout comes with it's own padding. If we were able to set a dark option on the docs page we could use the same mecahnism for both.